### PR TITLE
feat: add near-field occluded-emergence replay

### PR DESCRIPTION
**Pull request by ll7**
## Summary
Adds a #3323 near-field live replay matrix for the #2777 observation-noise wrapper while preserving the #2756 occluded-emergence first-visible/delay boundary. The promoted compact evidence now classifies the near-field one-seed run as `policy_insensitive`, not robustness evidence.

## Linked Issues
- Closes `#3323`
- Relates to `#2777`
- Follow-up: `#3328`

## Stack / Dependency
- Base dependency: `PR #3324`
- Required prior PRs: `#3324` (already merged)
- Stack follow-up issues: `#3328`
- Safe to review independently: yes
- Review dependency reason, if any: this PR builds on the #3324 live replay wrapper and fixture-boundary matrix.

## What Changed
- Added a tracked #3323 route override that places the robot in the right corridor for a near-field encounter with the #2756 `h1` pedestrian.
- Added a #3323 single scenario and scenario matrix that keep the loaded scenario identity as `issue_2756_occluded_emergence`, seed `111`, and fixture timing metadata.
- Added a focused wrapper test that verifies fixture contract preservation and the applied near-field route.
- Updated compact evidence and the evidence catalog with the new `policy_insensitive` result.

## Why It Matters
- Added value: removes the previous `scenario_too_weak` blocker for the #2777 live replay by producing a near-field trace.
- Expected impact: makes the observation-noise replay evidence more interpretable; perturbations now occur in a near-field geometry but still do not change selected commands or progress/risk summaries.
- Why this is worth merging now: it closes the scoped #3323 geometry gap and records the remaining robustness gap as #3328.

## Research Result Guidance
- Target claim / hypothesis / blocker this should affect: #3323 blocker that the #2777 live replay was too distant to test near-field occluded-emergence perturbations.
- Comparator or baseline, if applicable: prior #3320 evidence on the same fixture boundary, classified `scenario_too_weak`.
- Evidence tier: targeted smoke / diagnostic probe.
- Result classification: diagnostic-only / `policy_insensitive`.
- Decision or stop rule, if applicable: accept when fixture timing remains no-op step 5 and delay-only step 7, and no-op closest robot-pedestrian distance is within 2 m.
- Parent issue, claim map, registry, context note, or synthesis surface to update: `docs/context/evidence/issue_2777_live_observation_noise_replay/` and `docs/context/evidence/README.md`.
- New research/benchmark/metric/paper-facing analysis tool, if any: NA - no new analysis tool.

## Domain-Aware Approval
- Required for this PR: yes - evidence-sensitive diagnostic stress classification changes from `scenario_too_weak` to `policy_insensitive`.
- Domains reviewed: evidence classification, fixture-boundary validity, benchmark interpretation.
- Status: waived
- Approver/review source or waiver: agent self-review waiver for merge readiness because the PR keeps the claim boundary diagnostic-only, preserves fail-closed fixture checks, and opens #3328 for any stronger behavior-sensitive or robustness claim.
- Validity checklist:
  - Target claim/hypothesis: #3323 only resolves the near-field geometry blocker for #2777 live replay; it does not claim robustness.
  - Comparator or split/evidence validity: no-op replay is compared with six #2755 perturbation families on the same scenario, seed, candidate, and horizon.
  - Fallback/degraded exclusions: no fallback/degraded execution is counted as evidence; wrapper and raw trace assertions verify the fixture timing.
  - Claim boundary: one scenario, one seed, diagnostic stress-slice evidence; stronger behavioral interpretation is deferred to #3328.
  - Implementation integrity vs experimental validity: tests and PR readiness cover implementation; this approval only covers the evidence classification boundary.

## Falsification / Non-Transfer Check
- Did the mechanism activate? yes.
- Did the intervention change command source, selected command, trajectory, or route progress? The near-field geometry activated, and perturbations changed planner-input observations, but selected commands and progress/risk summaries stayed unchanged.
- Did the scenario actually contain the targeted failure mode? It contained the targeted near-field occluded-emergence geometry; no behavior-sensitive planner failure appeared in this one-seed run.
- Result route: continue via #3328 for behavior-sensitive or stronger evidence.
- Follow-up question or issue for weak, negative, or non-transfer results: #3328.

## Next Empirical Action
- Rerun needed: yes, for any stronger robustness or behavior-sensitive claim.
- Extractor or analysis tool needed: no.
- Artifact missing or unavailable: no for #3323 scope; raw traces remain worktree-local and compact evidence is tracked.
- Stop / revise / continue decision: stop the #3323 geometry fix; continue separately under #3328.
- Proposed child issue or existing follow-up: #3328.

## Validation / Proof
- Commands run:
  - `uv run pytest tests/benchmark/test_issue_2777_observation_noise_live_replay.py tests/validation/test_run_policy_search_step_diagnostics.py -q`
  - `uv run python scripts/benchmark/run_observation_noise_live_replay_issue_2777.py --scenario-matrix configs/scenarios/sets/issue_3323_occluded_emergence_near_field_live_replay.yaml --output-dir output/validation/issue-3323-live-replay-parent`
  - raw trace assertion script over `output/validation/issue-3323-live-replay-parent`
  - `uv run robot_sf_bench validate-config --matrix configs/scenarios/sets/issue_3323_occluded_emergence_near_field_live_replay.yaml`
  - `uv run ruff check tests/benchmark/test_issue_2777_observation_noise_live_replay.py`
  - `uv run ruff format --check tests/benchmark/test_issue_2777_observation_noise_live_replay.py`
  - `git diff --check`
  - `BASE_REF=origin/main uv run python scripts/dev/run_compact_validation.py -- scripts/dev/pr_ready_check.sh`
- Evidence that the change works here: live replay completed with `fixture_contract.satisfied=true`, no-op first observed step `5`, delay-only first observed step `7`, no-op closest robot-pedestrian distance `1.65520666531114`, and all six perturbed conditions classified `policy_insensitive`.
- Benchmarks or smoke tests, if applicable: targeted live replay and full PR readiness passed.

## Risks / Rollout
- Compatibility risks: low; the new matrix is additive and default #3320 matrix behavior is preserved.
- Failure modes: the evidence remains one scenario and one seed, so it must not be used as a robustness or planner-superiority claim.
- Rollback or fallback plan: revert the additive #3323 configs, focused test, and compact evidence update.

## Docs / Provenance
- Updated docs: `docs/context/evidence/README.md`, `docs/context/evidence/issue_2777_live_observation_noise_replay/README.md`.
- Relevant design or provenance notes: compact evidence records the matrix, seed, fixture timing, planner mode, and classification; raw traces are intentionally not committed.
- Any assumptions that need to be preserved: fallback/degraded execution is not benchmark evidence; this run is diagnostic stress evidence only.

## Downstream Propagation
- Parent issue updated (yes/no/NA): yes, via this PR closing #3323.
- Claim map / benchmark report updated (yes/no/NA): NA; not a benchmark-strength claim.
- Leaderboard / artifact catalog updated (yes/no/NA): artifact/evidence catalog updated.
- Registry or config index updated (yes/no/NA): new tracked scenario matrix/config files added; no separate registry update required.
- Context index / memory note updated (yes/no/NA): evidence catalog updated; broader memory note NA.
- Follow-up issue opened for deferred propagation (yes/no/NA): yes, #3328.
- Not applicable because: stronger robustness propagation is deferred to #3328.

## Follow-Up Issues
- Deferred work: behavior-sensitive or stronger near-field observation-noise replay evidence.
- Issues opened for follow-up: #3328.

## Reviewer Notes
- Verify the compact evidence remains diagnostic-only and does not overclaim robustness.
- The config validator reports expected one-seed/diagnostic comparability warnings for this stress slice.

## Summary by CodeRabbit

* **New Features**
  * Added near-field test scenarios for occluded-emergence edge cases with diagnostic stress testing.

* **Tests**
  * Added validation test for near-field route overrides and fixture contract preservation.

* **Documentation**
  * Updated diagnostic evidence documentation with refined classification metrics and closest robot-pedestrian distance data.

### CODE REVIEW

File: tests/benchmark/test_issue_2777_observation_noise_live_replay.py [hunk 1]
```diff
@@ -75,6 +79,45 @@ def test_issue_3320_matrix_satisfies_occluded_emergence_contract() -> None:
...
+    route_payload = yaml.safe_load(route_override.read_text(encoding="utf-8"))["route_payload"]
+    assert route_payload["robot_routes"] == [
+        {"spawn_id": 1, "goal_id": 1, "waypoints": [[27.0, 9.0], [27.0, 4.5]]}
+    ]
+
+    scenarios = load_scenarios(matrix)
+    assert len(scenarios) == 1
+    scenario = scenarios[0]
+    assert scenario["name"] == "issue_2756_occluded_emergence"
+    assert Path(str(scenario["route_overrides_file"])).resolve() == route_override
```
[the remark this answers is not in the document]
Comment (ll7) [reply tests/benchmark/test_issue_2777_observation_noise_live_replay.py]: Addressed in aedef81edbd77262fc7048ef89348a26789ebd22 by resolving route_overrides_file relative to the matrix path when needed instead of relying on cwd/module globals.

File: docs/context/evidence/issue_2777_live_observation_noise_replay/summary.json [hunk 2]
```diff
@@ -47,15 +47,10 @@
   "run_config": {
     "candidate": "risk_surface_dwa_v0",
     "stage": "smoke",
-    "scenario_matrix": "configs/scenarios/sets/issue_3320_occluded_emergence_live_replay.yaml",
-    "scenario_name": null,
-    "scenario_index": 0,
-    "seed": null,
-    "seed_index": 0,
+    "scenario_matrix": "configs/scenarios/sets/issue_3323_occluded_emergence_near_field_live_replay.yaml",
+    "seed": 111,
```
[the remark this answers is not in the document]
Comment (ll7) [reply docs/context/evidence/issue_2777_live_observation_noise_replay/summary.json]: Addressed in aedef81edbd77262fc7048ef89348a26789ebd22 by restoring the generated _run_config fields in the committed compact summary while preserving the tracked diagnostic evidence values.
